lib/countly_flutter.dart

```diff
@@ -9,15 +9,20 @@ class Countly {
   static const MethodChannel _channel =
   const MethodChannel('countly_flutter');
 
-
   // static variable
+  static Function listenerCallback;
   static bool isDebug = false;
   static bool enableCrashReportingFlag = false;
   static Map<String, Object> messagingMode = {"TEST": "1", "PRODUCTION": "0", "ADHOC": "2"};
   static Map<String, Object> deviceIDType = {
     "TemporaryDeviceID": "TemporaryDeviceID"
   };
 
+  static log(String message) async {
```

Comment:
If this call was added then it should be used. Replace all other
"
if(isDebug){
      print(args);
    }
"
calls with the 'log' call.